kolibri/core/assets/src/core-app/mediator.js

```diff
@@ -450,13 +450,27 @@ export default class Mediator {
       } else {
         // We have a content renderer for this, but it has not been loaded, so load it, and then
         // resolve the promise when it has been loaded.
-        Promise.all(this._contentRendererUrls[kolibriModuleName].map(this._scriptLoader))
-          .then(() => {
+        const urls = this._contentRendererUrls[kolibriModuleName].filter(
+          url =>
+            (languageDirection === languageDirections.RTL &&
+              url.includes(languageDirections.RTL)) ||
+            !url.includes('css')
+        );
+        Promise.all(urls.map(this.scriptLoader))
+          .then(scriptsArray => {
+            const storeTags = module => {
+              module.urlTags = {};
+              urls.forEach((url, index) => {
+                module.urlTags[url] = scriptsArray[index];
+              });
```

Comment:
Some more comments in here might be helpful. Without context, it's not clear to me what "store tags" and "url tags" are.